index.html

```diff
@@ -58,7 +60,221 @@ <h3>
 
 </section>
    
+
+<header class="title-header">
+    <h2 class="title">My Recent Works</h2>
+  </header>
+  <section class="recent-works">
+      <div class="work">
+          <section >
+  <img src="/img/Img-Placeholder.png" alt="">
```

Comment:
- [ ] All the images on your page are not showing. I think it's because the src is incorrect. You're missing the `.` it should look like this :
 `<img src="./img/Img-Placeholder.png" alt="">`
Kindly fix the issue in all the not showing images.